series.go

```diff
@@ -40,6 +52,7 @@ type Series struct {
 	Length      *int        `json:"length,omitempty"`
 	Scope       *string     `json:"scope,omitempty"`
 	Expression  *string     `json:"expression,omitempty"`
+	Units       *[2]*Unit   `json:"unit"`
```

Comment:
This isn't incredibly readable. I'd rather it be declare a UnitPair type that's an alias for a slice of Unit pointers and have this be a pointer to that type.